kuma/settings/common.py

```diff
@@ -940,15 +940,15 @@ def pipeline_one_scss(slug, **kwargs):
     },
     'react-main': {
         'source_filenames': (
-            'js/libs/jquery/jquery.js',
-            'js/libs/icons.js',
-            'js/components.js',
+            #'js/libs/jquery/jquery.js',
+            #'js/libs/icons.js',
+            #'js/components.js',
             'js/analytics.js',
-            'js/main.js',
-            'js/auth.js',
-            'js/highlight.js',
-            'js/wiki-compat-trigger.js',
-            'js/lang-switcher.js',
+            #'js/main.js',
+            #'js/auth.js',
+            #'js/highlight.js',
+            #'js/wiki-compat-trigger.js',
+            #'js/lang-switcher.js',
```

Comment:
I'm experimenting here to see how much JS I can strip out. I think I've got a bit too far: I'm seeing errors in the console about missing jquery in js/analytics.js.

(And come to think of it, that will become a fatal error in production when these modules are squashed into one, so I need to fix it...)